lib/array.rb

```diff
@@ -0,0 +1,9 @@
+class Array
+
+  def new_inject (a=self.shift, &block)
```

Comment:
Do not put a space between a method name and the opening parenthesis.<br>Surrounding space missing in default value assignment.